### PR TITLE
Warning message

### PR DESCRIPTION
**Pull request by tt-le**
Added warning message thrown by test case and explaining that it is expected.

### CODE REVIEW

**Review by AustinSeto — APPROVED** [review]
LGTM

File: README.adoc [hunk 1]
```diff
@@ -440,13 +440,18 @@ You will see the following output:
...
+Could not send Message.
+[err] The specified host is unknown: java.net.UnknownHostException: UnknownHostException invoking http://badhostname:9080/inventory/properties: badhostname
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.264 sec - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 
 Results :
 
 Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning message is expected and results from a request to a bad or an unknown host name.
```
Comment (gkwan-ibm) [inline README.adoc:453]: @tt-le 
- "The warning and error messages are ..."
- mention from the test `testUnknownHost()` of InventoryEndpointIT

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc:453]: and License year at the top should be updated to `2017, 2020`

**Review by griffinhadfield — APPROVED** [review]
Verified in lgdev. LGTM.

File: README.adoc [hunk 2]
```diff
@@ -440,13 +440,18 @@ You will see the following output:
...
+Could not send Message.
+[err] The specified host is unknown: java.net.UnknownHostException: UnknownHostException invoking http://badhostname:9080/inventory/properties: badhostname
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.264 sec - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 
 Results :
 
 Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning and error messages are expected and result from a request to a bad or an unknown host name. This request is made in the test `testUnknownHost()` from `InventoryEndpointIT`.
```
Comment (Charlotte-Holt) [inline README.adoc:453]: Suggested update

>The warning and error messages are expected and result from a request to a bad or an unknown hostname. This request is made in the `testUnknownHost()` test from the `InventoryEndpointIT` endpoint.

- I think endpoint works after "`InventoryEndpointIT`". Please update to the correct noun if I got it wrong. Thx!

Comment (gkwan-ibm) [reply README.adoc]: @Charlotte-Holt 
I updated to 
> The warning and error messages are expected and result from a request to a bad or an unknown hostname. This request is made in the `testUnknownHost()` test from the `InventoryEndpointIT` intergation test.
what do you think?

Comment (Charlotte-Holt) [reply README.adoc]: @gkwan-ibm Looks good. Just a typo in "intergation" - update to "integration".

Comment (gkwan-ibm) [reply README.adoc]: updated

**Review by Charlotte-Holt — APPROVED** [review]

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**